paddle/framework/ddim.cc

```diff
@@ -278,5 +278,11 @@ std::ostream& operator<<(std::ostream& os, const DDim& ddim) {
   return os;
 }
 
+ssize_t DDim::size() const { return product(*this); }
+
+DDim::DDim(std::initializer_list<int> init_list) {
```

Comment:
This constructor could be significantly easier for Op developer to invoke `Tensor::set_dims`.

Just by
```cpp
tensor.set_dims({100, 784})
```